specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -361,6 +381,58 @@ function MergeButton<SCHEMA extends AnySchema>({
   );
 }
 
+export function Status({
+  mergingId,
+  loadingBar: [_, setLoadingBar],
+}: {
+  readonly mergingId: string;
+  readonly loadingBar: GetSet<boolean>;
+}): JSX.Element {
+  const [status, setStatus] = React.useState<string>();
+  const [total, setTotal] = React.useState<number>();
+  const [current, setCurrent] = React.useState<number>();
+
+  React.useEffect(() => {
+    const fetchStatus = () =>
```

Comment:
why do you need to create a fetchStatus() function if you call it right away, and only in one place? just call ajax() right away, without creating a function